src/compiler/types.ts

```diff
@@ -3977,34 +3978,34 @@ namespace ts {
     }
 
     export const enum EmitFlags {
-        SingleLine = 1 << 0,                     // The contents of this node should be emitted on a single line.

```

Comment:
What's the point of the whitespace change here? It seems arbitrary how much to use.